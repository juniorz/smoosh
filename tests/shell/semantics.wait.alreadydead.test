sleep 10 &
pid=$!
sleep 1
kill $pid
echo kill ec: $?
sleep 1
wait $pid
echo wait ec: $?
