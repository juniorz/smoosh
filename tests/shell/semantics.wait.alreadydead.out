kill ec: 0
wait ec: 143
